generators/bdk/java/templates/ext-app/java/SecurityConfig.java.ejs

```diff
@@ -0,0 +1,17 @@
+package <%= basePackage %>;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        // This configuration is not recommended in production setup
```

Comment:
What is recommended for production setup? Is there a way we can avoid that?

Reply:
the extension app require the cross-origin config to work on client, so I only add the simplest security config which permit any request at the generated project.

Reply:
for this comment, I think we would keep the security config as simple like that and developer will configure their all security configurations.

Reply:
do we clearly understand what each line does here?
to me:

first one is to disable spring security on endpoints (by default if add the module it adds basic auth everywhere)

second, I'm not so sure why we need it, do we have such requests that would be prevented in an extension app?

third is to allow cross frame communication I guess, but could we suggest to restrict it to *.symphony.com for production? (although it looks deprecated and maybe https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy should be used instead)

as all of that is really not obvious commenting each line would be useful (to us and to users...)